content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -709,10 +748,13 @@ global:
     oidc:
       clientId: ""
       secret: ""
+      # Only include if your IdP requires it.
       authMethod: ""
       issuer: ""
 ```
 
+`authMethod` is optional. 
```

Comment:
The comment previously seems to imply that it is not optional, either your IdP requires it (it's not optional) or it doesn't (then does it even evaluate the field?)